mesheryctl/internal/cli/root/system/reset.go

```diff
@@ -39,6 +41,29 @@ var resetCmd = &cobra.Command{
 // resets meshery config
 func resetMesheryConfig() error {
 	log.Info("Meshery resetting...")
+
+	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+	if err != nil {
+		return errors.Wrap(err, "error processing config")
+	}
+
+	currentContext := mctlCfg.CurrentContext
+	currChannel := mctlCfg.Contexts[currentContext].Channel
+	currVersion := mctlCfg.Contexts[currentContext].Version
+
+	fileURL := ""
+
+	if currChannel == "edge" {
+		fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	} else if currChannel == "stable" {
+		fileURL = "https://raw.githubusercontent.com/layer5io/meshery/" + currVersion + "/docker-compose.yaml"
+	}
+
+	log.Printf("Current Context: %s", currentContext)
+	log.Printf("Current Channel: %s", currChannel)
+	log.Printf("Current Version: %s", currVersion)
+	log.Printf("File URL: %s", fileURL)
```

Comment:
@navendu-pottekkat, yes, to your question, this debug line can be removed.

Reply:
@leecalcote What should it output instead?